ParProcCo/passthru_wrapper.py

```diff
@@ -42,7 +46,13 @@ def create_slice_jobs(
             str(job_scheduling_information.job_resources.cpu_cores),
         )
         if job_scheduling_information.output_filename:
-            args += ("--output", job_scheduling_information.output_filename)
+            args += (
+                "--output",
+                str(
+                    job_scheduling_information.output_dir
+                    / job_scheduling_information.output_filename
```

Comment:
This could be simpler:
```
output_path = job_scheduling_information.get_output_path()
if output_path is not None:
    args += ("--output", str(output_path))

It's worth noting that if `job_scheduling_information.output_dir` is None, it returns `Path(self.output_filename)`
```

Reply:
Yep!